### ISSUE TITLE
Routing

### ISSUE THREAD

**Issue by IanPouncey** [bbc/gel-test-docs#12]

**Comment by IanPouncey** [comment]
## Extrapolation from GEL technical recommendations

### Required

* When a loading spinner is used it must be available to screen readers
* When a loading spinner is used the "Loading..." element must be focused
* The document `<title>` element content must reflect the current screen at all times

### Recommended

* Only the unique content for individual screens should change as the user is rerouted
* The unique screen content should be housed in a `<main>` element
* The `<main>` element should be accessible by keyboard using a skip link
* A skip link should be the first interactive element on the page, and allows keyboard users to bypass the header and navigation where desirable
* The `<title>` text should be made up of a label for the page and a label for the site or application

### Implied

* Use `aria-current="page"` on navigation links that correspond to the current page
* Set focus to the new content's `<h1>` elment when handling route changes
* Do not set focus when the application is first loaded (allow the browser default behaviour to apply)

## Suggested ACT expectations

* The document `<title>` element content must reflect the current screen at all times
* A skip link should be the first interactive element on the page
* Use `aria-current="page"` on navigation links that correspond to the current page
* Set focus to the new content's `<h1>` elment when handling route changes
* Do not set focus when the application is first loaded (allow the browser default behaviour to apply)

**Comment by micmath** [comment]
re "When a loading spinner is used the "Loading..." element must be focused"
This implies there is only ever one loading spinner at a time?

**Comment by micmath** [comment]
This is good to go.
